package.json

```diff
@@ -30,13 +30,13 @@
     "clean": "builder concurrent clean-lib clean-dist clean-es",
     "build-dist-min": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.js --colors",
     "build-dist-dev": "webpack --bail --config node_modules/builder-victory-component/config/webpack/webpack.config.dev.js --colors",
-    "build-dist": "builder run clean-dist && builder concurrent --queue=1 build-dist-min build-dist-dev",
+    "build-dist": "builder run clean-dist && builder concurrent --buffer build-dist-min build-dist-dev",
```

Comment:
@ryan-roemer we only talked about using `--buffer` in the `build` script, but `build-dist` and `build-libs` should be good-to-go as well